### ISSUE TITLE
Active class

### ISSUE THREAD

**Issue by Oksana13** [naveenkumarr05/weather-app#13]
There is no active class on tabs, it should be by requirements, users will be very confused without it

https://github.com/naveenkumarr05/weather-app/blob/89bbedf3d4fd693a7f39c6db777a1ad077478127/src/components/DayForecastView.vue#L3

**Comment by naveenkumarr05** [comment]
yeah Correct, As per requirement there is no active class on Tab.
